src/main/public/js/select-toggled-content.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function () {
+  $('.timeline-row .form-control-select').on('change', function () {
+
+    var $content = $(this).next()
+
+    if (this.value) {
+      $content
+        .removeClass('js-hidden')
+        .attr('aria-hidden', 'false')
+        .attr('aria-expanded', 'false')
```

Comment:
I do not think `aria-expanded` should be here. This tag is used to indicate state of the element what expands content, not on the content being expanded itself.